src/httprpc.cpp

```diff
@@ -204,6 +204,7 @@ static bool HTTPReq_JSONRPC(const std::any& context, HTTPRequest* req)
             }
 
             jreq.parse(valRequest);
+            if (jreq.IsNotification()) return false;
```

Comment:
In 94d02376c853aa4835de413d3f75b8b06bcb5193 "rpc: JSON-RPC 2.0 should not respond to "notifications""

This results in 500 and a response body of "Unhandled request". I think it should actually be a 200 with no response body.

```suggestion
            if (jreq.IsNotification()) {
                req->WriteReply(HTTP_OK);
                return true;
            }
```

Reply:
I'm looking into this, the [spec](https://www.jsonrpc.org/specification#notification) repeats no response to notifications but it doesn't indicate whether an HTTP OK is reasonable or whether the request should be completely ignored. I'm having some trouble getting libevent to completely cancel a request without sending anything back (and still letting bitcoind shutdown without hanging!) so maybe a 200 OK is best

Reply:
There's https://www.simple-is-better.org/json-rpc/transport_http.html  (linked from the wikipedia page) which says 202 or 204; there's also https://www.simple-is-better.org/json-rpc/extension_transport.html by the same author but dated two months earlier, which suggests 200 or 204.

Reply:
ah awesome thank you, I'll go with `204 No Response`

Reply:
Added in push to 26dcb359acf421dc3f448b49e0b564e714c54792. Also had to tweak `authproxy.py` to accommodate code 204